be/src/common/config.cpp

```diff
@@ -1401,7 +1401,7 @@ Status persist_config(const std::string& field, const std::string& value) {
     // lock to make sure only one thread can modify the be_custom.conf
     std::lock_guard<std::mutex> l(custom_conf_lock);
 
-    static const std::string conffile = std::string(getenv("DORIS_HOME")) + "/conf/be_custom.conf";
+    static const std::string conffile = config::custom_config_dir + "/be_custom.conf";
```

Comment:
if user does not set config::custom_config_dir, where to save the be_custom.conf??